config/express.js

```diff
@@ -65,11 +65,11 @@ passport.deserializeUser(function(id, done) {
 module.exports.http = {
 
   customMiddleware: function(app) {
-
     passport.use(new RedditStrategy({
       clientID: Reddit.data.clientID,
       clientSecret: Reddit.data.clientIDSecret,
-      callbackURL: Reddit.data.redirectURL
+      callbackURL: Reddit.data.redirectURL,
+      scope: 'flair,modflair,modcontributors,wikiread,wikiedit,read,modposts'
```

Comment:
Oh I see what you did. You set the default to be everything, and override that for normal users? Interesting.